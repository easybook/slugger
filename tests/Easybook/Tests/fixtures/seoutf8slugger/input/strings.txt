ذرزسشصضطظعغػؼؽؾؿ 5.99€
ذرزسشصضطظعغػؼؽؾؿ £5.99
׆אבגדהוזחטיךכלםמן $5.99
日一国会人年大十二本中長出三同 ¥5990
5.99€ 日一国会人年大十 $5.99
בגדה@ضطظعغػ.com
年大十@ضطظعغػ
בגדה & 年大十
国&ם at ضطظعغػ.הוז
my username is @בגדה
The review gave 5* to ظعغػ
use 年大十@ضطظعغػ.הוז to get a 10% discount
日 = הط^2
ךכלם 国会 غػؼؽ 9.81 m/s2
The #会 comment at @בגדה = 10% of *&*