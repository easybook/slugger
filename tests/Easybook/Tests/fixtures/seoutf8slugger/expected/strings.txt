dhrzsshsdtzgh-5-euros-99-cents
dhrzsshsdtzgh-5-pounds-99-pence
bgdhwzhtykklmmn-5-dollars-99-cents
ri-yi-guo-hui-ren-nian-da-shi-er-ben-zhong-zhang-chu-san-tong-5990-yen
5-euros-99-cents-ri-yi-guo-hui-ren-nian-da-shi-5-dollars-99-cents
bgdh-at-dtzgh-dot-com
nian-da-shi-at-dtzgh
bgdh-and-nian-da-shi
guo-and-m-at-dtzgh-dot-hwz
my-username-is-at-bgdh
the-review-gave-5-star-to-zgh
use-nian-da-shi-at-dtzgh-dot-hwz-to-get-a-10-percent-discount
ri-equals-ht2
kklm-guo-hui-gh-9-dot-81-m-s2
the-number-hui-comment-at-at-bgdh-equals-10-percent-of-star-and-star